webrtc.html

```diff
@@ -9566,7 +9578,39 @@ <h3>RTCPeerConnection Interface Extensions</h3>
       <div>
         <pre class="idl">partial interface RTCPeerConnection {
     Promise&lt;RTCStatsReport&gt; getStats (optional MediaStreamTrack? selector = null);
-};</pre>
+    attribute EventHandler onstatsended;
+          };</pre>
+        <section>
+          <h2>Attributes</h2>
+          <dl data-link-for="RTCPeerConnection"
+              data-dfn-for="RTCPeerConnection" class="attributes">
+            <dt><dfn><code>onstatsended</code></dfn> of type
+              <span class="idlAttrType"><a>EventHandler</a></span></dt>
+            <dd>
+              <p>The event type of this event handler
+                is <code><a>statsended</a></code>.
+              </p>
+              <p>When the lifetime of one or more <a>monitored object</a>s ends, the UA MUST
+                queue a task to run the following steps:
```

Comment:
Can you clarify that "if the same event causes multiple monitored objects' lifetime to end, they MUST be added to the same report." Alternatively, "any monitored objects' ending in the same event loop execution cycle must be fired in the same event" but that phrasing would require different algorithmic steps.

I think there is room for interpretation here, but a UA that implemented queuing one such task per ended object would hit the issues previously discussed.

Reply:
+1 for the clarification. Additionally,  this should be covered in the webrtc-stats spec in the lifetime of the objects.

Reply:
Changed to defining a "delete stats" operation, which can be linked to from other places in the spec.